src/tailwind.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
         0: "0",
         xs: "18rem",
       },
+      maxHeight: {
+        0: "0",
+        full: "1000px",
+      },
```

Comment:
full by right is `100%`, if this is gonna be used 1 time only, i'm rather it being inline style instead of declaring here. separately i understand that tw v2 will have max-h-0, look forward to that

Reply:
are we going to upgrade to tw-v2?

Reply:
I tried 100% it doesn't do the transition thus I will change it to 100vh instead